### ISSUE TITLE
urn:cts:latinLit:phi1017.phi012 is a sum of works

### ISSUE THREAD

**Issue by PonteIneptique** [PerseusDL/catalog_data#61]
I am not totally sure about this but from what I understand, http://catalog.perseus.org/catalog/urn:cts:latinLit:phi1017.phi012 covers more than one work, which would mean we need to cut it in many other works.

**Comment by AlisonBabeu** [comment]
Actually the Dialogi of Seneca have been divided up into other works under their STOA ID, this is a case where the PHI had only one ID for the entire work but the STOA directory had assigned all of the dialogues individual IDs, so we used their STOA IDs.  All of the Dialogi within Perseus have been cataloged under the STOA ID and all of the URNs have corresponded. For example, one of the Dialogi, De Ira, on Perseus http://www.perseus.tufts.edu/hopper/text?doc=Perseus:text:2007.01.0014 has the CTS-URN: urn:cts:latinLit:stoa0255.stoa010.perseus-lat1. All of the works that make up the Dialogi have CTS URNs using stoa0255 and have been aggregated as well under that textgroup:
http://catalog.perseus.org/catalog/urn:cts:latinLit:stoa0255.

Because all of the records also had the PHI ID attached to them it also created records for them there.

**Comment by PonteIneptique** [comment]
So, then I'd like to ask : what is the real textgroup for Seneca ? It seems weird to have two textgroup for one author... I ask that before I am currently converting Seneca and it would be probably a good time to group everything in a way or another.

**Comment by AlisonBabeu** [comment]
I'm not sure if we could call on the real textgroup so much as that we decided to use PHIs as the preferred ID for Latin authors that had them, STOA as a secondary ID for those same authors for works without PHIs, and we've exclusively used STOA for new Latin authors moving forward. Some authors like Cicero actually have three textgroups when they wrote in Latin and Greek, http://catalog.perseus.org/catalog/urn:cite:perseus:author.364.  The authority records have served to group all the records with different IDs under one author.

**Comment by balmas** [comment]
I will forever regret the decision to use tlg and phi identifiers for our textgroups.

I think this is one of those times when implementation highlights issues with decisions made in the abstract.  From the perspective of a CTS repository, having two textgroups for a single author in this case doesn't really make alot of sense.  

Are there are any works catalogued under the Phi textgroup that aren't represented in the Stoa textgroup?

**Comment by AlisonBabeu** [comment]
Lots of authors (probably several hundred), unfortunately, since STOA didn't create identifiers for fragmentary authors while the PHI did. The PHI also created identifiers for anonymous works.

**Comment by balmas** [comment]
Oh, I was just asking about Seneca specifically right now. Looking at the catalog, it seemed to me that we only have the dialogs catalogued under the phi1017 textgroup. If that's the case, then we could only use the stoa identifiers in our CTS 5 inventory I think (and maybe redirect the phi identifiers in the catalog)

**Comment by AlisonBabeu** [comment]
There is only one PHI identifier for Seneca's Dialogi, this was one of those cases where Perseus had broken up the works and had used ABO identifiers. Once we had the STOA identifier list we decided to go with those instead. There are actually many other orks cataloged under the phi1017 textgroup(http://catalog.perseus.org/catalog/urn:cts:latinLit:phi1017) as well.

**Comment by balmas** [comment]
oh .. so I actually have the problem reversed. the only works catalogued under the stoa identifier are the dialogs.

I guess what we probably should have done was to mint new work identifiers for the dialogs, under the PHI textgroup.

So I think this is actually a use case for our new DTS working group... we need an ability in our new model to aggregate textgroups.

**Comment by AlisonBabeu** [comment]
Unfortunately this is not the only place where this happened in the catalog either where we had to do strange things with identifiers in order to aggregate works, but I'll leave it at that.

**Comment by balmas** [comment]
yes, that's why I think it's a use case for DTS  - we jumped through hoops to get CTS to work where it didn't

**Comment by AlisonBabeu** [comment]
Sounds good to me, just one question, what does DTS stand for?

**Comment by balmas** [comment]
distributed text services https://distributedtext.slack.com
